src/utils/user/TodoApiService.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { User, Login} from "../../interface/userInterface";
+
+export function registerUser(user: User): any{

```

Comment:
use `any` as response type is not a good practice. 

Consider introducing a response type as below:

```suggestion
export function registerUser(user: User):  RegisterUserResponse{

```